environment/bashrc/.bashrc

```diff
@@ -217,6 +215,6 @@ if [[ "${verbose:=false}" == "true" ]]; then
   echo "in draco/environment/bashrc/.bashrc ... done";
 fi
 
-##---------------------------------------------------------------------------##
-## end of .bashrc
-##---------------------------------------------------------------------------##
+#---------------------------------------------------------------------------#
+# end of .bashrc
+#---------------------------------------------------------------------------#
```

Comment:
Minor: should these lines also go to 100 columns? :)

Reply:
Fixed by 7b03549